validation/policies/io/konveyor/forklift/openstack/vm_image_based.rego

```diff
@@ -0,0 +1,14 @@
+package io.konveyor.forklift.openstack
+
+image_based_vm {
+	input.imageID != ""
+}
+
+concerns[flag] {
+	image_based_vm
+	flag := {
+		"category": "Critical",
+		"label": "Image based VMs are not supported",
+		"assessment": "The VM is image based, only volume based VMs are supported. Attempting to migrate this VM may fail.",
```

Comment:
I'm not sure about the last part - if it's critical then we shouldn't say that the migration may succeed, how about
```suggestion
		"assessment": "The VM is image based and migration of VMs that are based on images is not supported. Only migration of VMs that are based on volumes is supported.",
```

Reply:
What about `The VM is 'Image' based which is not currently supported. Only the migration 'Volume' based VMs is supported. The VM can be migrated but it will not boot.`?

Reply:
> the first two parts are ok (just missing "of": `The VM is 'Image' based which is not currently supported. Only the migration of 'Volume' based VMs is supported`). 
I will fix that missing `of`.

>about the last part - wouldn't this validation, as a critical one, block the migration?
Indeed, I will remove that last part.

Reply:
Done